program/src/processor.rs

```diff
@@ -3008,6 +2947,15 @@ impl Processor {
 
         let market_index = mango_group.find_perp_market_index(perp_market_ai.key).unwrap();
 
+        // Owner signature not necessary if market is in ForceClose
+        let mode = mango_group.tokens[market_index].perp_market_mode;
+        if mode == MarketMode::ForceCloseOnly {
+            // To guard against possibility delegate is zero key and that's passed in
+            check!(owner_ai.key != &Pubkey::default(), MangoErrorCode::InvalidOwner)?;
```

Comment:
I don't understand this check, can you explain? `owner_ai` is either the owner or the delegate. Why is passing Pubkey::default() bad?

Reply:
You're right this check is not needed